iOS/FVM/Framework/Logging/Log.swift

```diff
@@ -25,6 +25,8 @@ internal class Log {
     }
     private static var dateFormat = "yyyy-MM-dd hh:mm:ss"
     
+    private init() {}
+    
```

Comment:
Can we still use this class?

Reply:
Yes, we don't need ctor, it has only static methods